src/Files.App/Data/Items/PageOnStartupItem.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.App.Data.Items
+{
+	public class PageOnStartupItem
+	{
+		public string Text
+			=> ShellHelpers.GetShellNameFromPath(Path);
+
+		public string Path { get; }
+
+		internal PageOnStartupItem(string path)
+			=> Path = path;
```

Comment:
I don't know if this counts, but we shouldn't use single line arrow functions

Reply:
I should not do that. Done